include/cpp-statsd-client/StatsdClient.hpp

```diff
@@ -84,13 +84,16 @@ class StatsdClient {
 
     //! The random number generator for handling sampling
     mutable std::mt19937 m_randomEngine;
+
+    //! The buffer string format our stats before sending them
+    mutable std::string m_buffer;
 };
 
 inline StatsdClient::StatsdClient(const std::string& host,
                                   const uint16_t port,
                                   const std::string& prefix,
                                   const uint64_t batchsize) noexcept
-    : m_prefix(prefix), m_sender(new UDPSender{host, port, batchsize}) {
+    : m_prefix(prefix), m_sender(new UDPSender{host, port, batchsize}), m_buffer(256, '\0') {
```

Comment:
256 seems big enough to me but maybe sometime in the future we could make it configurable